diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3795323..b54a07f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -17,8 +17,12 @@
 
 cmake_minimum_required(VERSION 3.6)
 
-include(MunkeiVersionFromGit.cmake)
-version_from_git(TIMESTAMP "%Y%m%d%H%M%S")
+set( GIT_TAG       @git_tag@       )
+set( SEMVER        @semver@        )
+set( VERSION       @version@       )
+set( VERSION_MAJOR @version_major@ )
+set( VERSION_MINOR @version_minor@ )
+set( VERSION_PATCH @version_patch@ )
 
 project(tuxedo-touchpad-switch
         VERSION ${VERSION}
